src/ai/ai.hpp

```diff
@@ -137,8 +137,10 @@ class AI {
 
 	/**
 	 * Get the number of days before the next AI should start.
+	 * @param count Current count of number of AI company indexes that are planned to be starting.
```

Comment:
I wasn't ignoring (or maybe I was). I just couldn't figure if your comment would suffice. Pretty vague statement, so that's why I didn't say anything.